google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurable.java

```diff
@@ -82,24 +75,11 @@ public JComponent createComponent() {
 
   @Override
   public boolean isModified() {
-    // Forces a modify check so the user is unable to save an invalid Cloud SDK configuration from
-    // Other Settings, on the Clouds menu.
-    return cloudSdkPanel.isModified()
-        || !CloudSdkService.getInstance().isValidCloudSdk(cloudSdkPanel.getCloudSdkDirectoryText());
+    return cloudSdkPanel.isModified();
   }
 
-  /**
-   * Users shouldn't be able to save a cloud configuration without a valid Cloud SDK configured.
-   */
   @Override
-  public void apply() throws ConfigurationException {
-    String message = cloudSdkPanel.buildSdkMessage(cloudSdkPanel.getCloudSdkDirectoryText(),
-        false /*htmlEnabled*/);
-
-    if (!StringUtil.isEmpty(message)) {
-      throw new RuntimeConfigurationError(message);
-    }
-    
+  public void apply() {
     if (cloudSdkPanel != null) {
```

Comment:
Actually I took out this null check since if it were null, something would be pretty wrong with IntelliJ since it populates this field from the form.